src/connectors/hits-per-page/__tests__/connectHitsPerPage-test.js

```diff
@@ -19,7 +19,7 @@ describe('connectHitsPerPage', () => {
       ],
     });
 
-    expect(widget.getConfiguration).toEqual(undefined);
+    expect(typeof widget.getConfiguration).toEqual('function');
 
```

Comment:
Since we have a `getConfiguration` we should test the returned value of this function.